src/class_addressGenerator.py

```diff
@@ -135,8 +140,8 @@ def run(self):
                         potentialPubSigningKey + potentialPubEncryptionKey)
                     ripe = RIPEMD160Hash(sha.digest()).digest()
                     if (
-                        ripe[:numberOfNullBytesDemandedOnFrontOfRipeHash] ==
-                        '\x00' * numberOfNullBytesDemandedOnFrontOfRipeHash
+                            ripe[:numberOfNullBytesDemandedOnFrontOfRipeHash] ==
+                            '\x00' * numberOfNullBytesDemandedOnFrontOfRipeHash
```

Comment:
why?

Reply:
Pylint warning: Wrong hanging indentation before a block.

Reply:
What version? So how do you think I merged it?

Reply:
I'm currently using pylint 1.9.5. Maybe checks using an old version of pylint.

Reply:
https://github.com/PyCQA/pylint/issues/289

Reply:
It applies to all such expressions. I removed extra spaces intentionally.